composer.json

```diff
@@ -17,17 +17,19 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "friendsofsymfony/oauth2-php": "~1.1.0",
-        "symfony/framework-bundle": "~2.2",
-        "symfony/security-bundle": "~2.1"
+        "friendsofsymfony/oauth2-php": "~1.1",
```

Comment:
@Ener-Getick seems like you already released oauth2-php 1.2 compatible with symfony/http-foundation 3?